counter-app/src/Buttons.tsx

```diff
@@ -1,6 +1,6 @@
-import React from "react";
```

Comment:
It is not necessary to `import React` for projects which have architecture such as this one.